packages/tools/webpack/extract-i18n-expressions-webpack-plugin.js

```diff
@@ -1,47 +1,39 @@
-const { sources: { RawSource } } = require( 'webpack' );
+const fs = require( 'fs' );
+const path = require( 'path' );
 
 const MODULE_FILTERS = Object.freeze( [ /(([^!?\s]+?)(?:\.js|\.jsx|\.ts|\.tsx))$/, /^((?!node_modules).)*$/ ] );
 
-module.exports = class ExtractI18nExpressionsWebpackPlugin {
-	translationsRegexps;
-	generateTranslationFilename;
-
-	constructor( {
-		// WordPress i18n function, example for regex match: `__('Hello', 'elementor')`, `_n('Me', 'Us', 2, 'elementor-pro')`.
-		translationsRegexps = [ /\b_(?:_|n|nx|x)\(.*?,\s*(?<c>['"`])[\w-]+\k<c>\)/ ],
-		generateTranslationFilename,
-	} = {} ) {
-		if (
-			! translationsRegexps ||
-			! Array.isArray( translationsRegexps ) ||
-			translationsRegexps.some( ( regexp ) => ! ( regexp instanceof RegExp ) )
-		) {
-			throw new Error( '`translationsRegexps` must be an array of RegExp' );
-		}
+const COMMENTS_REGEXPS = Object.freeze( [
+	// `/* translators: %s is a placeholder for the name of the plugin. */`.
+	/\/\*[\t ]*translators:.*\*\//gm,
+	// `// translators: %s is a placeholder for the name of the plugin.`.
+	/(\/\/)[\t ]*translators:[^\r\n]*/gm,
+] );
 
-		this.generateTranslationFilename = generateTranslationFilename || this.defaultGenerateTranslationFilename;
-		this.translationsRegexps = translationsRegexps.map( ( regex ) => {
-			const flags = [ ...new Set( [ 'g', 'm', ...regex.flags.split( '' ) ] ) ].join( '' );
-
-			return new RegExp( regex.source, flags );
-		} );
-	}
+const TRANSLATIONS_REGEXPS = Object.freeze( [
+	// `__('Hello', 'elementor')`, `_n('Me', 'Us', 2, 'elementor-pro')`.
```

Comment:
```suggestion
	// Matches translation functions: `__('Hello', 'elementor')`, `_n('Me', 'Us', 2, 'elementor-pro')`.
```

just so it won't look like you forgot a commented code here

Reply:
Done